test/validatorUtils.test.ts

```diff
@@ -8,29 +8,9 @@ import { Deposit } from "../src/interfaces";
 
 describe("validatorUtils", () => {
   describe("isDeposit", () => {
-    let deposit: interfaces.V2DepositWithBlock, depositV3: interfaces.V3DepositWithBlock;
```

Comment:
`V3DepositWithBlock` hasn't been defined for quite some time. Why did this not error before :face_exhaling:

Reply:
This is strange. Should we bump the strictness for TSC in the tests?